### PR TITLE
Add description and HTML color to BJCP style

### LINKED ISSUE #29: Add rendered SRM and field descriptions to BeerStyle model/serializer
Copy the info from #28 into the BeerStyle model.

### PR DESCRIPTION
**Pull request by drewbrew**
Title says basically everything.

Fixes #29. Depends on #28.

### CODE REVIEW

File: beers/models.py [hunk 1]
```diff
@@ -38,18 +40,45 @@ class BeerStyle(models.Model):
...
-    ibu_high = models.PositiveSmallIntegerField(default=0)
-    srm_low = models.PositiveSmallIntegerField(default=0)
-    srm_high = models.PositiveSmallIntegerField(default=0)
+    ibu_low = models.PositiveSmallIntegerField(
+        'Minimum bitterness (International Bitterness Units)', default=0,
+    )
+    ibu_high = models.PositiveSmallIntegerField(
+        'Maximum bitterness (International Bitterness Units)', default=0,
+    )
+    srm_low = models.PositiveSmallIntegerField(
```
Comment (mjcarroll) [inline beers/models.py]: SRM is an integer here and Decimal in Beer.

File: beers/models.py [hunk 2]
```diff
@@ -38,18 +40,45 @@ class BeerStyle(models.Model):
...
+    fg_high = models.DecimalField(
+        'Maximum final specific gravity',
+        max_digits=4, decimal_places=3, default=0,
+    )
 
-    abv_low = models.DecimalField(max_digits=3, decimal_places=1, default=0)
-    abv_high = models.DecimalField(max_digits=3, decimal_places=1, default=0)
+    abv_low = models.DecimalField(
+        'Maximum alcohol content (% by volume)',
+        max_digits=3, decimal_places=1, default=0,
```
Comment (mjcarroll) [inline beers/models.py]: Doesn't match definition in beer

File: beers/models.py [hunk 3]
```diff
@@ -38,18 +40,45 @@ class BeerStyle(models.Model):
...
+    )
+    abv_high = models.DecimalField(
+        'Minimum alcohol content (% by volume)',
+        max_digits=3, decimal_places=1, default=0,
```
Comment (mjcarroll) [inline beers/models.py]: Doesn't match definition in beer

File: beers/models.py [hunk 4]
```diff
@@ -80,3 +115,58 @@ class Manufacturer(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Beer(models.Model):
+    name = models.CharField(max_length=25, db_index=True)
```
Comment (mjcarroll) [inline beers/models.py]: Now that I think about it, I actually think that 25 could be too short.  Especially given the crazy names as well as collaboration beers that are out there now.

File: beers/serializers.py [hunk 5]
```diff
@@ -84,3 +96,51 @@ class ManufacturerSerializer(serializers.ModelSerializer):
...
+        write_only=True, required=True, allow_null=False,
+        queryset=models.Manufacturer.objects.all(),
+    )
+    abv = serializers.DecimalField(
+        max_digits=4, decimal_places=2, min_value=0,
+        max_value=Decimal('99.99'), allow_null=True, required=False,
+    )
+    color_srm = serializers.DecimalField(
+        max_digits=4, decimal_places=1, min_value=1,
+        # 500 SRM = the darkest specialty grain currently available
```
Comment (mjcarroll) [inline beers/serializers.py]: DARKEST NIGHT

File: beers/serializers.py [hunk 6]
```diff
@@ -84,3 +96,51 @@ class ManufacturerSerializer(serializers.ModelSerializer):
...
+
+class BeerSerializer(serializers.ModelSerializer):
+    manufacturer = ManufacturerSerializer(read_only=True)
+    manufacturer_id = serializers.PrimaryKeyRelatedField(
+        write_only=True, required=True, allow_null=False,
+        queryset=models.Manufacturer.objects.all(),
+    )
+    abv = serializers.DecimalField(
+        max_digits=4, decimal_places=2, min_value=0,
+        max_value=Decimal('99.99'), allow_null=True, required=False,
```
Comment (mjcarroll) [inline beers/serializers.py]: Gimme dat 99.99 beer.

File: beers/test/test_utils.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+    def test_negative(self):
+        self.assertEqual('#ffffff', render_srm(-1).lower())
+
+    def test_above_30(self):
+        self.assertEqual(render_srm(30), render_srm(5000))
+
+    def test_not_int(self):
+        with self.assertRaises(ValueError):
+            render_srm('🦐')
```
Comment (mjcarroll) [inline beers/test/test_utils.py]: Shrimp is not an `int`.  Had Python been statically typed, this test would have been unnecessary.

File: beers/models.py (hunk 2 above)
Comment (drewbrew) [reply beers/models.py]: I talked about that in the other PR. Monkeynaut is 7.25% in Untappd, but the BJCP styles only go to tenths of a percent.

File: beers/models.py (hunk 1 above)
Comment (drewbrew) [reply beers/models.py]: If the brewers use the calculated SRM value, then it'll be decimal. If they're doing it against reference samples in a lab, then it'll be an integer.

Have you seen a data source that actually gives us SRM?

File: beers/test/test_utils.py (hunk 7 above)
Comment (drewbrew) [reply beers/test/test_utils.py]: You say unnecessary, I say far less fun.

File: beers/models.py (hunk 4 above)
Comment (drewbrew) [reply beers/models.py]: I'm good with extending to 50.

Comment (mjcarroll) [reply beers/models.py]: So maybe Unicode as well?

* Stone Enjoy By 01.01.19 Brüt IPA

And finding longer beer names: 

* Cigar City  => Marshal Zhukov's - Apple Brandy Barrel Aged W/ Madagascar Vanilla Beans (71)

File: beers/models.py (hunk 2 above)
Comment (mjcarroll) [reply beers/models.py]: Oh yeah, fine by me then.

File: beers/models.py (hunk 1 above)
Comment (mjcarroll) [reply beers/models.py]: Uhhhh. No. I have not.

File: beers/models.py (hunk 4 above)
Comment (drewbrew) [reply beers/models.py]: Unicode is already taken care of for free.

File: beers/models.py (hunk 1 above)
Comment (drewbrew) [reply beers/models.py]: I think for now we can leave it alone and then migrate if needed later.

File: beers/models.py (hunk 4 above)
Comment (drewbrew) [reply beers/models.py]: Extended to 100 characters

**Pull request merged**